src/components/Page.vue

```diff
@@ -41,6 +41,9 @@
           <li class="nav-item" v-if='modules.user_tasks'>
           <router-link class="nav-link" :to='{"name": "page.user_tasks", params: { group: group, cluster: cluster } }'>Cruise Control Tasks</router-link>
           </li>
+          <li class="nav-item" v-if='modules.user_tasks'>
```

Comment:
should this `modules.user_tasks` be changed to `charts.$something` ?

Reply:
good point, I propose `charts.enabled` (with a new `charts` object in the store, that will maybe contain more data later on, should we need to centralize more config about the charts). What do you think?

Reply:
How do you prefer for me to proceed with the fixes, should I push fix commits?

Reply:
looks we have modules.chart_page now. closing